dart/src/main/java/com/f2prateek/dart/internal/ExtraInjector.java

```diff
@@ -189,4 +189,16 @@ private void emitFieldBindings(StringBuilder builder, ExtraInjection injection)
       }
     }
   }
+
+  public Map<String,ExtraInjection> getInjectionMap() {
+    return injectionMap;
+  }
+
+  public String getClassPackage() {
```

Comment:
it comes from the variable name in `ExtraInjector`. I didn't want to change the existing code.